learning/tour-of-beam/frontend/lib/pages/tour/widgets/content.dart

```diff
@@ -0,0 +1,115 @@
+/*
```

Comment:
The widgets in this folder were extracted from the screen file. All issues here, if any, are not a regression.